src/pfe/portal/modules/User.js

```diff
@@ -291,7 +291,7 @@ module.exports = class User {
       for (let i = 0; i < projectArray.length; i++) {
         let project = projectArray[i];
         // Don't request build if autoBuild is false
-        if (project.isOpen() && project.autoBuild) {
+        if (project.isOpen()) {
```

Comment:
As far as I can tell, this change means that even if the user has `autoBuild` turned off, the projects will `buildAndRun` when PFE starts. Is that intended?